src/netedit/elements/demand/GNEVType.cpp

```diff
@@ -850,6 +857,8 @@ GNEVType::isValid(SumoXMLAttr key, const std::string& value) {
             return canParse<double>(value) && (parse<double>(value) >= -1);
         case SUMO_ATTR_CARRIAGE_GAP:
             return canParse<double>(value) && (parse<double>(value) >= 0);
+        case SUMO_ATTR_DIRECT_TURN_PROBABILITY:
+            return canParse<double>(value) && (parse<double>(value) >= 0);
```

Comment:
You can also add here a condition to check value <= 1.0

Reply:
Please revisit